loader/readsource.go

```diff
@@ -67,6 +67,11 @@ func ReadSource(
 	pwdURL := &url.URL{Scheme: "file", Path: filepath.ToSlash(filepath.Clean(pwd)) + "/"}
 	srcURL, err := Resolve(pwdURL, filepath.ToSlash(src))
 	if err != nil {
+		var noSchemeError noSchemeRemoteModuleResolutionError
+		if errors.As(err, &noSchemeError) {
+			// TODO maybe try to wrap the original error here as well, without butchering the message
```

Comment:
I copied this from the other place this is made, below, but looking at it ... I think it is done?